theme_loftspace/views/snippets/s_banner.xml

```diff
@@ -4,16 +4,48 @@
 <template id="s_banner" inherit_id="website.s_banner">
     <!-- Section -->
     <xpath expr="//section" position="attributes">
-        <attribute name="class" add="o_cc o_cc4" separator=" "/>
+        <attribute name="class" add="o_cc o_cc5" remove="pt96 pb96" separator=" "/>
     </xpath>
-    <!-- Title -->
-    <xpath expr="//h1" position="replace" mode="inner">
-        The art of Design
+    <!-- Container -->
+    <xpath expr="//div[hasclass('container')]" position="attributes">
+        <attribute name="class" add="container-fluid" remove="container" separator=" "/>
     </xpath>
-    <!-- Paragraph -->
-    <xpath expr="//p" position="replace" mode="inner">
-        Our products are designed for small to medium size companies willing to optimize their performance.
+    <!-- Row -->
+    <xpath expr="//div[hasclass('row')]" position="attributes">
+        <attribute name="data-row-count" add="20" remove="10" separator=" "/>
```

Comment:
Sorry... but what about setting it to 20 instead of add+remove ? This is not a composite value.